.github/workflows/main.yml

```diff
@@ -172,83 +167,3 @@ jobs:
         run: |
           pytest tests/test_main_system_caches.py
           pytest -m 'vetiver'
-
-  test-connect:
-    needs: distributions
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        PY_VERSION:
-          - 3.11.13
-          - 3.12.11
-    steps:
-          - uses: extractions/setup-just@v2
-            env:
-              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          - uses: actions/checkout@v4
-            with:
-              fetch-depth: 0
-            env:
-              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          # Checkout the rsconnect-python tests from Connect
-          - uses: actions/checkout@v4
-            with:
-              repository: rstudio/connect
-              path: 'test/connect-rsconnect-python'
-              sparse-checkout: |
-                  test/rsconnect-python
-                  tools/dev
-                  examples
-              sparse-checkout-cone-mode: false
-              token: ${{ secrets.CONNECT_PAT }}
```

Comment:
Are you removing all uses of this secret? If so, let's delete it after this merges.